geometry/cellid.hpp

```diff
@@ -12,8 +12,9 @@ namespace m2
 template <int kDepthLevels = 31> class CellId
 {
 public:
-  // TODO: Move CellId::DEPTH_LEVELS to private.
-  static int const DEPTH_LEVELS = kDepthLevels;
+  // Use enum to avoid linker errors.
+  // Can't realize why static const or constexpr is invalid here ...
+  enum { DEPTH_LEVELS = kDepthLevels };
```

Comment:
Linker errors are due to non-odr usage of DEPTH_LEVELS. Just declare the constant here and define in the cpp file. Already discussed that :)